core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -2625,6 +2627,49 @@ class KafkaApis(val requestChannel: RequestChannel,
     }
   }
 
+  def handleOffsetDeleteRequest(request: RequestChannel.Request): Unit = {
+    val offsetDeleteRequest = request.body[OffsetDeleteRequest]
+    val groupId = offsetDeleteRequest.data.groupId()
+
+    if (authorize(request.session, Read, Resource(Group, groupId, LITERAL))) {
```

Comment:
Yeah. I thought it makes more sense to allow an application to delete an offset if it can commit it. On the other hand, we could argue that deleting is more an exceptional behaviour so it could make sense to have the possibility to restrict it. I think the later is more reasonable. I will update it.